packages/suite/src/views/settings/SettingsGeneral/Tor.tsx

```diff
@@ -57,11 +58,7 @@ export const Tor = () => {
     return (
         <SettingsSectionItem anchorId={SettingsAnchor.Tor}>
             <TextColumn
-                title={
-                    <LoadingContent isLoading={isTorLoading} isSuccessful={!hasTorError}>
```

Comment:
Why not keep this? It would make sense to show a loading indicator here in my opinion

Reply:
That was they way it was working before, so it was loading and when Tor was ready it was marked as enabled but if the process of starting Tor and establishing a successful Tor circuit was failing for some reason (for instance timeout) the loading was completed and Tor was marked as disabled. 

With this PR the concept is changing to have this toggle to work just as user expression to use Tor, so it Tor fails it starts again until user manually disable it. This is in order to avoid disable Tor when user actually wants to use it.

There is plan to add more information about the Tor status in different ways in the UI.

Reply:
Yes I see the change, but I still think it's useful to keep this so the user can clearly see it's loading. 
When I come to the settings and turn Tor on, it's more obvious to see that something is happening right there, than looking at the small Tor status icon in the bottom left.

Reply:
Yes, I know what you mean and I agree, I would add there in the setting toggle below or something more information about the Tor status, and why it is taking more time and so on. But it looks we will address that in future improvement.

Reply:
Ok got it